.babelrc

```diff
@@ -1,5 +1,6 @@
 {
   "stage": 1,
   "optional": ["runtime"],
-  "loose": ["all"]
+  "loose": ["all"],
+  "plugins": ["object-assign"]
```

Comment:
that is a good question, at one point it didn't polyfill Object.assign, may not be true now tho...